README.md

```diff
@@ -378,14 +378,31 @@ easier.
 You should not collect results intended for publication with development switches
 turned on.
 
-We also recommend that for 'real' benchmarking you turn on the
-`--hardware-reboots` and `--daemonise` switches. These ensure that the system
-will reboot before each benchmark execution, and that Krun will run in the
-background, allowing you to log out before the first reboot.
-
-You will also need to ensure that Krun is restarted once the machine has
-rebooted. The `etc/` directory contains example `/etc/rc.local` files for the
-platforms supported by Krun.
+We also recommend that benchmarking in "reboot mode". The recommended way to do
+this is via `cron(8)`. We supply a script `start_krun_from_cron` to make this
+easier.
+
+As a regular user (who has access to your experiment and the Krun code), invoke
+`crontab -e` and add and add lines similar to the following:
+
+```
+CONFIG_FILE=your_experiment.krun
+KRUN_DIR=/path/to/krun/dir
+EXPERIMENT_DIR=/path/to/dir/containing/config/file
+@reboot /usr/bin/env /bin/sh -c ${KRUN_DIR}/scripts/start_krun_from_cron
+```
+
+If you wish to pass custom flags to Krun, you can also add `EXTRA_KRUN_ARGS` to
+the environment.
+
+You can then start the experiment via a regular invocation of Krun with
```

Comment:
Huh? Isn't `start_from_krun` supposed to do this for you?

Reply:
I suppose you could invoke the script directly, but that wasn't my initial intention.

Reply:
Let me test it works and get back to you If it does, then I'd actually prefer this, as it takes less explanation in the docs.

Reply:
Sounds good!